snippets/ossm-out-of-support.adoc

```diff
@@ -0,0 +1,14 @@
+// Text snippet included in all Service Mesh v1 assemblies.
+// NOTE: The OpenShift docs standards state that snippets should NOT contain xrefs.   https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-text-snippets
+//Because this snippet contains two xrefs it should ONLY be used in the v1 assemblies and never in a module.
+
+:_content-type: SNIPPET
+
+[WARNING]
+====
+*You are viewing documentation for a {SMProductName} release that is no longer supported.*
+
+Service Mesh version 1.0 and 1.1 control planes are no longer supported. For information about upgrading your service mesh control plane, see xref:../../service_mesh/v2x/upgrading-ossm.adoc#ossm-versions_ossm-upgrade[Upgrading Service Mesh].
+
+For information about the support status of a particular {SMProductName} release, see the https://access.redhat.com/support/policy/updates/openshift#ossm[Product lifecycle page].
```

Comment:
Please update this to use `link`, as follows:

> For information about the support status of a particular {SMProductName} release, see the link:https://access.redhat.com/support/policy/updates/openshift#ossm[Product lifecycle page].

Reply:
File removed.